core/web/resolver/eth_key.go

```diff
@@ -81,7 +81,7 @@ func (r *ETHKeyResolver) LINKBalance(ctx context.Context) *string {
 
 	client := r.key.chain.Client()
 	addr := common.HexToAddress(r.key.chain.Config().LinkContractAddress())
```

Comment:
nit: rename addr to linkAddr. Makes it clear which one is link address, and which is from address